metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '2.0.2'
+version           '2.0.3'
```

Comment:
This seems like a 3.0 release since poise changes a lot of how the underlying service management works